src/PlaywrightSharp/BrowserContextOptions.cs

```diff
@@ -273,5 +273,16 @@ public BrowserContextOptions Clone()
 
             return args;
         }
+
+        private BrowserContextOptions CopyFrom(BrowserContextOptions options)
```

Comment:
If this is private we could just assign the values in the constructor, what do you think?

Reply:
No, because the way it works right now is, it'll return the `options` instance, not the current one. I didn't want to mess with that behaviour too much *yet*, until I actually get around to `IBrowserContext` and related classes.